packages/upstream/conf-gmp.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "nbraud"
+author: "nbraud"
+homepage: "http://gmplib.org/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+license: "GPL"
+build: [
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "darwin"}
+  ["sh" "-exc" "cc -c $CFLAGS -I/opt/local/include -I/usr/local/include test.c"] {os = "darwin"}
+]
+depexts: [
+  [["debian"] ["libgmp-dev"]]
+  [["ubuntu"] ["libgmp-dev"]]
+  [["osx" "homebrew"] ["gmp"]]
+  [["centos"] ["gmp" "gmp-devel"]]
```

Comment:
Do we have GMP installed on xenserver hosts?

Reply:
It seems that `gmp` is installed on a xenserver:
```
# rpm -q gmp
gmp-6.0.0-12.el7_1.x86_64
```